### ISSUE TITLE
Don't show the delete button when clicking "Select all matching"

### ISSUE THREAD

**Issue by wfeller** [laravel/nova-issues#322]
The "Delete selected" option is shown when clicking "Select all matching" in the resource index, even when the user is actually not allowed to delete anything (see images below).

Is there a way to avoid this behaviour please? Maybe adding a deleteAny() method on the resource's policy?

Thanks

![image](https://user-images.githubusercontent.com/21337627/44802940-52bbad00-abbd-11e8-8a8e-5878dd111bca.png)

![image](https://user-images.githubusercontent.com/21337627/44802961-5e0ed880-abbd-11e8-8f53-1fcccbc8c48e.png)

**Comment by dddeeemmmooonnn** [comment]
Still not fixed

**Comment by mmanzano** [comment]
Same problem here.

**Comment by IgnacioPerezMolina** [comment]
We are waiting the fix of this issue too.

A possible solution could be creating a template to overwrite the original template, but I think that it isn't the best solution for it.

**Comment by davidhemphill** [comment]
Hey there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.

**Comment by mmanzano** [comment]
Sorry about that, Laravel have not a repo for feature requests. I am searching about a forum where I can request this feature. I published about that on Slack and Discord nova channels. Where is the place for a feature request?

**Comment by dddeeemmmooonnn** [comment]
This is bug, why feature request? @davidhemphill

**Comment by rayoplateado** [comment]
I think this could be related... https://twitter.com/Rayoplateado/status/1105362708008980480

**Comment by disitec** [comment]
This is a bug. Why the UI is showing a delete button if user can not delete the record? If you click in it NOTHING HAPPENS.
This is not a feature and if you don't have time to matain a project you should think about it.

**Comment by dkulyk** [comment]
Nova cannot check permissions when you check "Select All Matching" therefore Nova shows delete menu.

Adding `deleteAny` permission is feature request.

**Comment by disitec** [comment]
For me is a feature needed for solving a bug. But anyways you decide. But clients need their apps working. Just saying.

Thanks.

**Comment by boghy933** [comment]
Hello guys,

If someone is still searching for a solution i know this is not perfect, but for now i just hide the menu using CSS.

```css
div[dusk="delete-menu"] {
    display: none !important;
}
```

**Comment by bobbybouwmann** [comment]
@boghy933 solution is the only once that actually blocks this process... Can we get a fix for this?

**Comment by cord** [comment]
@crynobone this is a bug

the nova resource policy

```
    public function authorizedToDelete(Request $request)
    {
        return false;
    }
```

is ignored when clicking "Select all matching" - otherwise, it is applied the delete button remains hidden

**Comment by cord** [comment]
> Hey there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.

@davidhemphill please reopen as this definitely is a functional bug

**Comment by crynobone** [comment]
@cord Please submit a new issue with full example to reproduce the issue. 

> We don't reopen 2 year olds issue when the discussion may no longer be relevant with the current status of Nova.

**Comment by cord** [comment]
@crynobone just reposted - in my case I have a read-only view on log-data with an export feature. In this setup the delete route is disabled for all items.
